Update Companion App Documentation and Update Core Concepts Testing

tmhastings opened the pull request:
- Comprehensive details on Trio's integration with:
    - LoopFollow
    - Nightscout
    - xDrip4iOS
- Addition of an "Explain It To Me" Section to explain difficult or confusing concepts that are staples in the Trio app or ecosystem. Currently includes pages on:
    - Loop Cycle Pill (in Trio and Nightscout)
    - Desmos Graphs
    - Using Dynamic ISF (relocated from "Features")
- Fixes broken links
- Updates Index and mkdocs pages
- Updates Glossary 
- Relocates Companion Apps section from Install to Usage
- Relocates Migration Guide section from Configuration to Usage

tmhastings committed:
Adds Nightscout documentation

tmhastings committed:
Merge branch 'nightscout:dev' into missing-services

tmhastings committed:
Updates to migration, remote control, loop follow, nightscout, an xdrip pages

Adding various updates to the docs after local system crash. This is to avoid loss of any additional edits.

These changes will most likely be edited more in the near future.

tmhastings committed:
Merge branch 'missing-services' of https://github.com/tmhastings/trio-docs into missing-services

tmhastings committed:
Update Companion App Pages

tmhastings committed:
Merge branch 'nightscout:dev' into missing-services

tmhastings committed:
Update Core Concepts pages

Add instructions for testing in closed loop

tmhastings committed:
Merge branch 'missing-services' of https://github.com/tmhastings/trio-docs into missing-services

tmhastings committed:
Broken Links Fixed

tmhastings committed:
Move Using Dynamic ISF to "Explain it to me" section

Moves Using Dynamic ISF to the Explain It To Me section and fixes broken links created by the move.

tmhastings committed:
Add a Desmos Graphs Explanation page

MikePlante1 commented on docs/install/ecosystem/nightscout-app.md:
```diff
@@ -1,2 +1,560 @@
+    - Device status
+    - Glucose readings
+    - Insulin delivery
+    - Carbs
```
+`, Fats, Proteins`

MikePlante1 commented on docs/install/ecosystem/index.md:
```diff
@@ -8,8 +8,8 @@ Please use the navigation menu :fontawesome-solid-bars:, search bar :fontawesome
--   __[Loop Follow ✏️](loop-follow.md)__
--   __[XDrip4iOS ✏️](xdrip4ios.md)__
+-   __[Nightscout](nightscout-app.md)__
+-   __[Loop Follow](loop-follow.md)__
```
no space: LoopFollow

MikePlante1 commented on mkdocs.yml:
```diff
@@ -273,9 +282,9 @@ nav:
-          - Loop Follow ✏️: install/ecosystem/loop-follow.md
-          - XDrip4iOS ✏️: install/ecosystem/xdrip4ios.md
+          - Nightscout: install/ecosystem/nightscout-app.md
+          - Loop Follow: install/ecosystem/loop-follow.md
```
No space: LoopFollow

MikePlante1 committed:
Fix names and change carbs to meals

MikePlante1 committed:
Fix NS Device Status

MikePlante1 commented on docs/install/ecosystem/nightscout-app.md:
```diff
@@ -1,2 +1,548 @@
+</div>
+
+!!! note
+    Glucose Readings are controlled by the "Upload Glucose" setting (can be disabled independently)
```
This note seems out-of-place.

MikePlante1 commented on docs/install/ecosystem/nightscout-app.md:
```diff
@@ -1,2 +1,548 @@
+- Carb amount (grams)
+- Fat and protein content (if entered)
+- FPU (Fat Protein Units) equivalents
+- Absorption notes
```
What are `absorption notes`?

MikePlante1 commented on docs/install/ecosystem/nightscout-app.md:
```diff
@@ -1,2 +1,548 @@
+- Fat and protein content (if entered)
+- FPU (Fat Protein Units) equivalents
+- Absorption notes
+- Timestamps (created and scheduled)
```
Not sure what `(created and scheduled)` means. As far as I can tell, NS just displays the time the carb was entered for.

MikePlante1 committed:
Update Nightscout documentation

Revised sections to clarify meal entries, temporary targets, overrides, and therapy profile data uploaded to Nightscout. Removed redundant notes and the local Nightscout usage instructions for improved clarity and relevance.

MikePlante1 committed:
Fix anchor links

tmhastings committed:
Update nightscout-app.md

tmhastings committed:
Merge branch 'nightscout:dev' into missing-services

tmhastings committed:
Relocate Companion Apps from Install to Usage

Relocates Companion Apps section from Install to Usage

Fixes broken links

tmhastings committed:
Revert "Relocate Companion Apps from Install to Usage"

This reverts commit 41d04dfb6ce36a6f71c790c2921ed0b3da24dc43.

tmhastings committed:
Relocate Companion Apps from Install to Usage

Relocates Companion Apps section from Install to Usage

Fixes broken links resulting from the move

tmhastings committed:
Adds Missing Sigmoid Variables in Menu

tmhastings committed:
Relocate Migration Guides to Usage instead of Configuration

Relocate Migration Guides from Configuration to Usage

Fix broken links caused by the move

tmhastings committed:
Return App Diagnostics to index file and mkdocs.yml

MikePlante1 committed:
Merge branch 'dev' into pr/207

MikePlante1 committed:
Revert "Update README.md"

This reverts commit 4160ce1c8e258042229dba3c503f3ea3dd3bad39.

MikePlante1 committed:
Remove unnecessary info and minor formatting

MikePlante1 committed:
Edit Nightscout Download section

MikePlante1 committed:
Add missing punctuation

MikePlante1 committed:
Fix link

MikePlante1 committed:
Update links

MikePlante1 committed:
Redirect old migration page links to new location

MikePlante1 committed:
Rename loop-follow.md to loopfollow.md and update references

MikePlante1 committed:
Add IAGE, SAGE, and TDD to glossary

MikePlante1 committed:
Update LoopFollow page

MikePlante1 committed:
Merge branch 'dev' into missing-services

MikePlante1 committed:
Merge branch 'dev' into missing-services

Natolida commented on docs/usage/ecosystem/loopfollow.md:
```diff
@@ -0,0 +1,328 @@
+### Data Display
+
+LoopFollow shows:
+
```
Could we include an image of the LoopFollow homescreen here to accompany the description below? 

A nicety but not necessary 🤷‍♂️

Natolida commented on docs/configuration/settings/features/remote-control.md:
```diff
@@ -1,38 +1,10 @@
 
-!!! warning "*Nightscout* version must be 15.0.2 or newer"
-    To properly display the OpenAPS pill with Trio 0.5.x (or newer), your *Nightscout* version must be 15.0.2 (or newer). If you do not see the expected treatments or pills in the *Nightscout* dashboard, follow the steps to [Configure for OpenAPS](#configure-for-openaps).
+The recommended approach is to use Trio 0.5.x (or newer) with the *LoopFollow* app to use the Trio Remote Control setting.
```
Consider updating to 0.7.x now that dev has been merged into main.

Natolida commented on docs/install/index.md:
```diff
@@ -11,7 +11,7 @@ Please use the navigation menu :fontawesome-solid-bars:, search bar :fontawesome
 !!! important "Attention Trio Users"
     At this time all users should [build the dev branch](build/browser/build-dev-with-browser.md) rather than the main branch. We are in the process of merging dev into main. Once that is complete, you will be able to build main again.
```
This needs updating. Build main over dev.

Natolida commented on docs/usage/ecosystem/xdrip4ios.md:
```diff
@@ -0,0 +1,382 @@
+1. Open **Trio** app
+2. Navigate to **Settings → Devices → CGM**
+3. Tap **Add CGM**
+4. Select **xDrip4iOS** from the list
```
Is this correct? With some Dexcom sensors via xdrip, you still select the specific CGM as the source in trio.

Natolida reviewed (changes requested):
Thats a long one 🫨

See a few comments throughout. I'll have to have another look through when my brain isnt so fatigued 😄
